pkg/operator/flags.go

```diff
@@ -61,7 +61,7 @@ func DefaultOperatorFlags() map[string]string {
 		MachineSetEnabled:                  FlagTrue,
 		MachineHealthCheckEnabled:          FlagTrue,
 		MachineHealthCheckManaged:          FlagTrue,
-		MonitoringEnabled:                  FlagTrue,
+		MonitoringEnabled:                  FlagFalse,
```

Comment:
This name sucks.  I wonder if we should keep the string but rename the var.  Your choice if you want to leave this as is.